flutter/news/test/features/frontpage/data/datasource/articles_remote_data_source_test.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http/http.dart' as http;
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:news/core/error/failures.dart';
+import 'package:news/core/result.dart';
+import 'package:news/features/frontpage/data/datasource/articles_remote_data_source.dart';
+import 'package:news/features/frontpage/data/models/article_model.dart';
+import 'package:news/features/frontpage/data/models/base_news_response_model.dart';
+
+import '../../../../core/fixtures/fixture_reader.dart';
+import 'articles_remote_data_source_test.mocks.dart';
+
+@GenerateMocks([http.Client])
+void main() {
+  late ArticlesRemoteDataSource remoteDataSource;
+  late MockClient mockHttpClient;
+
+  final testResult = Result.success(
+      BaseNewsResponseModel.fromJson(json.decode(fixture("articles.json")))
+          .articles as List<ArticleModel>);
+
+  setUp(() {
+    mockHttpClient = MockClient();
+    remoteDataSource = ArticlesRemoteDataSource(client: mockHttpClient);
+  });
+
+  test(
+    'Given Get request is done When doing the request Then the endpoint '
+    'being used should be /v2/top-headlines',
+    () async {
+      // arrange
```

Comment:
:) I am sure you wouldn't be adding this comments after reading `clean code`, 

There is a chapter about comments and Uncle Bob mentions there `Don't add obvious noise.`

Reply:
agree, have them on my custom template for tests and forgot to remove.

updated on [34d2f99](https://github.com/novoda/multiplatform-playground/pull/15/commits/34d2f99e34fbfc03733e882fdcc758565e92553f)